features/order/managing_orders/browsing_orders/sequential_order_number_generation.feature

```diff
@@ -17,8 +17,8 @@ Feature: Sequential order number generation
         And the customer chose "Free" shipping method to "United States" with "Offline" payment
         And I am logged in as an administrator
 
-    @ui
+    @ui @api
     Scenario: Verifying that orders has correct numbers
         When I browse orders
-        Then I should see an order with "#00000001" number
-        And I should see an order with "#00000002" number
+        Then I should see an order with "#000000001" number
```

Comment:
Just out of curiosity, why?

Reply:
UI assert ignores the missing `0`. In fact, the order number looks like the new version :P.